packages/table/src/regions.ts

```diff
@@ -528,6 +528,11 @@ export class Regions {
         return regionGroups;
     }
 
+    public static regionsEqual(regionA: IRegion, regionB: IRegion) {
```

Comment:
why move this? the previous location seems more logical to me: next to all the other comparison checks.